puter/log.py

```diff
@@ -38,7 +38,7 @@ def commit(data, stack):
         "meta": {"stack": f"{stack}", "datetime": timestamp},
     }
     log_path = puter.data_dir() / "log.txt"
-    log_path.touch(mode=0o600)
+    # log_path.touch(mode=0o600)
```

Comment:
Not sure why this was here, but it made it difficult to test -- I don't think it did anything that `open` does not, but maybe I'm wrong.

Reply:
I believe that was a result of a google rabbit hole into how to securely juggle files.

From memory: if you use touch like that, it ensures that the file has the designated permissions at all times. If you just rely on open (which I think can also take permissions to set on file), the file gets created, then the permissions get applied. Leaving a (tiny) gap of time where the file could be accessed by others.